screening/modular-open-source-identity-plaform.json

```diff
@@ -0,0 +1,130 @@
+      "stepsToPreventHarm": "Yes",
+      "additionalInfoMechanismProcessesPolicies": "MOSIP has a set of guidelines it adheres to while working with a country who is willing to adopt MOSIP. These guidelines also typically form part of the MOUs we enter into. The principles of engagement can be found here: https://www.mosip.io/uploads/resources/5cc84b0a08284Country%20Engagement%20Principles_v2.pdf",
+      "additionalMitigationSteps": [
+        "https://www.mosip.io/uploads/resources/5cc84b0a08284Country%20Engagement%20Principles_v2.pdf"
+      ]
+    },
+    "dataPrivacySecurity": {
+      "collectsPII": "Yes",
+      "typesOfDataCollected": [
+        "MOSIP enables collection of demographic and biographic information of the end user for the puspose of ascertaining and issuing a digital identity. The collection of demographic information are based on the project needs - and is configurable by the project owner. It typicaly contains:",
+        "- legal name",
+        "- age",
+        "- address and additional fields are collected as needed",
+        "Biometric information is collected for the purpose of ascertaining uniqueness and for authentiation, based on countries' policy:",
+        "- fingerprint",
+        "- face",
+        "- iris data is captured one time during the enrollment process."
+      ],
+      "thirdPartyDataSharing": "Yes",
+      "dataSharingCircumstances": [
+        "with user content, demographic data is shared for KYC as credentials for service. This  happens within a governance framework where the project owner mandates policies for sharing of eKYC data with service providers. Biometric information never leaves the system, except for facial images."
+      ],
+      "ensurePrivacySecurity": "Yes",
+      "privacySecurityDescription": "Privacy and security practices are central to MOSIP and the proejct has taken extensive measures to provide security of data and has numerous existing and evolving features on privacy and data protection. MOSIP's technological approach to security and privacy can be found here : https://docs.mosip.io/platform/architecture/privacy-and-security."
+    },
+    "inappropriateIllegalContent": {
+      "collectStoreDistribute": "No",
+      "type": "",
+      "contentFilter": "",
+      "policyGuidelinesDocumentationLink": "",
+      "illegalContentDetection": "",
+      "illegalContentDetectionMechanism": ""
+    },
+    "protectionFromHarassment": {
+      "userInteraction": "No",
+      "addressSafetySecurityUnderageUsers": "No",
+      "stepsAddressRiskPreventSafetyUnderageUsers": [
+        ""
+      ],
+      "griefAbuseHarassmentProtection": "Yes",
+      "harassmentProtectionSteps": [
+        "Link to code of conduct - https://docs.mosip.io/platform/contribute/code-of-conduct",
+        "MOSIP discussion groups and chat are moderated by team members for objectionable content - https://groups.io/g/mosip-dev"
+      ]
+    }
+  },
+  "locations": {
+    "developmentCounties": [
```

Comment:
My understanding is that MOSIP is being developed in India, so I would expect to see `India` listed here. Kindly confirm and edit accordingly.

Reply:
It appears India was not included by the contact person who submitted, kindly see [link](
https://docs.google.com/spreadsheets/d/1_DOubdcMeS1wMJO7P71OjTLKy9xbGD1L11FjObwd9iU/edit?ts=5fab4f5e#gid=1642917441) for more clarity on this question on line 115:

Reply:
I understand that they did not include it in their submission, but based on what we know about the project, I believe that is an oversight that we should correct. Can you confirm with them as you gather information from the other missing questions? Thanks 🙏

Reply:
Liv confirmed the institute behind MOSIP has its headquarters in India, so I will update the development countries to include India.

Reply:
Thanks 🙏